docs/stdlib/array.rst

```diff
@@ -130,9 +127,9 @@ Reference
 
 .. eql:operator:: arrayidx: array<anytype> [ int64 ] -> anytype
 
-    Array indexing.
+    Indexes an array of :eql:type:`anytype`.
```

Comment:
I'm not sure this provides much clarity. Suggestion:
> Accesses the element of the array at the given index (zero-indexed).

I think with that, you could maybe even drop the next sentence since the description will already tell you you're going to get back the thing at the index you specified.

Reply:
The main problem I run into with trying to simplify everything on the first paragraph is the 79 character limit. I'll likely have to reword the first paragraph to follown something similar, otherwise this will have to be the 2nd paragraph.

Reply:
Can you use the line continuation character to go beyond 79 characters if you need to? I'm pretty new to rST/Sphinx, so forgive me if that's not possible.

If you end up needing something shorter, this might work just as well:

> Accesses the array element at the index (zero-indexed)